src/dmd/target.d

```diff
@@ -478,7 +484,7 @@ struct Target
      */
     extern (C++) static LINK systemLinkage()
     {
-        return global.params.isWindows ? LINKwindows : LINKc;
+        return global.params.isWindows ? LINK.windows : LINK.c;
```

Comment:
This is why `LINK.system` feels like a hack. Should `systemLinkage` be called everywhere instead of using `LINK.system`?

Reply:
Hmm `LINK.system` is only set once by the parser:

https://github.com/wilzbach/dmd/blob/089241b1359d1a20f19dd1b589ff0922854b16bc/src/dmd/parse.d#L2208-L2211